packages/mermaid/src/rendering-util/rendering-elements/shapes/filledCircle.ts

```diff
@@ -0,0 +1,57 @@
+import { log } from '$root/logger.js';
+import { getNodeClasses, updateNodeBounds } from './util.js';
+import type { Node } from '$root/rendering-util/types.d.ts';
+import type { SVG } from '$root/diagram-api/types.js';
+import {
+  styles2String,
+  userNodeOverrides,
+} from '$root/rendering-util/rendering-elements/shapes/handDrawnShapeStyles.js';
+import rough from 'roughjs';
+import intersect from '../intersect/index.js';
+import { getConfig } from '$root/config.js';
+
+export const filledCircle = (parent: SVG, node: Node) => {
+  const { labelStyles, nodeStyles } = styles2String(node);
+  node.label = '';
+  node.labelStyle = labelStyles;
+  const shapeSvg = parent
+    .insert('g')
+    .attr('class', getNodeClasses(node))
+    .attr('id', node.domId ?? node.id);
+  const radius = 7;
+  const { cssStyles } = node;
+
+  // @ts-ignore - rough is not typed
+  const rc = rough.svg(shapeSvg);
+  const { themeVariables } = getConfig();
```

Comment:
Blocking:

Please add a 3rd parameter to the function, to which we pass config as a field. It should be an object containing config, so that we can add more optional parameters easily. 
We are planning to remove the use of getConfig to allow parallel rendering soon. So adding more calls should be avoided. 

```suggestion
export const filledCircle = (parent: SVG, node: Node, { config: { themeVariables } }: { config?: MermaidConfig } = {} ) => {
  const { labelStyles, nodeStyles } = styles2String(node);
  node.label = '';
  node.labelStyle = labelStyles;
  const shapeSvg = parent
    .insert('g')
    .attr('class', getNodeClasses(node))
    .attr('id', node.domId ?? node.id);
  const radius = 7;
  const { cssStyles } = node;

  // @ts-ignore - rough is not typed
  const rc = rough.svg(shapeSvg);
```

Reply:
This will require update in all shapes. None of them are updated.

Reply:
Yes, we should standardize the function signature for the shapes, so that it's consistent across shapes. Shape specific settings can be another parameter.

Reply:
Updated shapes to pass config